provider/elastictranscoder/aws.go

```diff
@@ -285,9 +285,18 @@ func (p *awsProvider) JobStatus(job *db.Job) (*provider.JobStatus, error) {
 			Width:    aws.Int64Value(resp.Job.Input.DetectedProperties.Width),
 		}
 	}
+	statusMessage := ""
+	if len(resp.Job.Outputs) > 0 {
+		statusMessage = aws.StringValue(resp.Job.Outputs[0].StatusDetail)
+		if strings.Contains(statusMessage, ":") {
+			errorMessage := strings.Split(statusMessage, ":")[1]
```

Comment:
The entire error looks like this:

`4000 b107230c-1f2a-4e8d-a2d9-3709612c91fb: Amazon Elastic Transcoder could not interpret the media file.`

So with the split and trimp above I get:

`Amazon Elastic Transcoder could not interpret the media file`